x/mint/simulation/store.go

```diff
@@ -0,0 +1,25 @@
+package simulation
```

Comment:
why put mint decoder in `x/mint/simulation` whereas gov decoder is in `x/gov/simulation/decoder`. Should be consistent approaches imo